src/backends/interflop-ieee/printf_specifier.c

```diff
@@ -0,0 +1,182 @@
+#include <endian.h>
+#include <math.h>
+#include <printf.h>
+#include <stdint.h>
+#include <stdio.h>
+
```

Comment:
Is this code yours ? If not please indicate the origin and licence.
Could you also please add a header ?

Reply:
Yes, it is. You're talking about a license header? If so, we should add one in `interflop_ieee.c` also.

Reply:
Yes indeed :-)